### ISSUE TITLE
Implement Polynomial learning rate decay

### ISSUE THREAD

**Issue by meetshah1995** [meetshah1995/pytorch-semseg#9]
Implement poly learning rate policy as described in [PSPNet](https://arxiv.org/pdf/1612.01105.pdf)

`lr = base_lr  *  (1 −iter / maxiter ) ^ power`

With base learning rate set to 0.01 and power set to 0.9

**Comment by meetshah1995** [comment]
done by @ibadami in PR #14
